src/course-outline/CourseOutline.scss

```diff
@@ -11,3 +11,21 @@
 @import "./drag-helper/SortableItem";
 @import "./xblock-status/XBlockStatus";
 @import "./paste-button/PasteButton";
+
+div.row:has(> div > div.highlight) {
+  animation: 5s glow;
+}
+
+@keyframes glow {
+  0% {
+    box-shadow: 0 0 5px 5px $primary-500;
+  }
+
+  90% {
+    box-shadow: 0 0 5px 5px $primary-500;
+  }
+
+  100% {
+    box-shadow: unset;
+  }
+}
```

Comment:
This is the animation after scrolling to a searched component. Personally, I didn't like it. :disappointed: 
I am open to suggestions!
![section-highlight](https://github.com/openedx/frontend-app-course-authoring/assets/849463/a5e84179-e766-4447-8491-8fbd95bf78bc)

![subsection-highlight](https://github.com/openedx/frontend-app-course-authoring/assets/849463/5bac982c-0624-4d7e-8e61-baaf18c45b3c)

![unit-highlight](https://github.com/openedx/frontend-app-course-authoring/assets/849463/2ea05032-076b-4ebf-ad91-fe19d85c042b)

Reply:
@rpenido I agree that the scrolling is awkward.. but I can't think of a better place to send someone who's searched for a Section/Subsection.

However, if they search for a Unit, I think we should direct them to the Unit edit page instead of scrolling down the outline, because Units are very unlikely to be expanded on that page. So then it's just Sections and Subsections that use the scrolling thing, and Units and components redirect to the Unit edit page.

@ali-hugo What do you think?

Reply:
The Unit redirect will be fixed in a follow-up task!

Reply:
What seems to be the issue with the animation? I think one thing that can be done to make it smoother is to use a timing function. 

i.e. remove the 90% step and add a timing function like:
`animation-timing-function: cubic-bezier(1,0,.72,.04);`

This will be a lot smoother I think since it'll follow a curve rather than a steep fall.

Reply:
> What seems to be the issue with the animation? 

Nothing specific. I just didn't find it "wow" (I'm not sure if it was the color, the glow size..). I think it's a subjective feeling, and I'm open to options to improve it (if that's the case).

Changed the curve here: 12ba0c5bd36921be99e214b0758adcae99a74bce

![new-animation](https://github.com/openedx/frontend-app-course-authoring/assets/849463/5cbf3387-1911-4037-ae38-967eadccf6af)